### DISCUSSION TITLE
Connect-CMS 定期勉強会のネタについて

### DISCUSSION THREAD

**Discussion by nagasheep** [opensource-workshop/connect-cms#682] [Ideas]
こんにちは。
株式会社オープンソース・ワークショップの永原です。

Connect-CMS 定期勉強会のネタについて、アイデアなど、ここで共有、募集してみようかなと思って書いてみます。

いま、聞いている要望
・XAMPP で環境を作る方法
・レンタルサーバーにインストールする方法
・NetCommons2 からの移行方法
他にもネタはいろいろあると思うので、メモしていきたいと思います。

定期勉強会としては、2021年1月20日（水）18:00から20:00 が最初になります。
最初はレンタルサーバーにインストールする方法なんてできたらなと思っていますが、まだ、私がレンタルサーバーにインストールしたことがないので（笑）、これから調査です。

以上

**Comment by dreamer9999** [comment]
NetCommons3 からの移行方法も話題に上げてほしいです。

**Reply by nagasheep** [reply]
NetCommons3 の「お知らせ」からある程度、移行できるプログラムは作ったのですが、まだ実験的なので、いま動くかは怪しいですね。
NetCommons3 からの移行はまだ先になりそうです。

**Comment by syunke** [comment]
プラグインの作り方教えてほしいです。

**Reply by nagasheep** [reply]
まとめきれてはいないのですが、勉強会で、ざっくりお話しすることはできると思います。

**Comment by nagasheep** [comment]
こんにちは。
株式会社オープンソース・ワークショップの永原です。

Connect-CMS勉強会の申込フォーム、予定する内容などの情報を公開しました。

2021年1月20日（水）18:00-20:00（Zoom）にて開催

Connect-CMS勉強会の申込フォームは以下
https://connect-cms.jp/forum

予定する内容などの情報は以下
https://connect-cms.jp/forum/schedule

皆様のご参加をお待ちしております。

**Comment by akagane99** [comment]
勉強会休止中のため、クローズさせて頂きます。
何かありましたら、再オープンして頂ければと思います。
